content/en/docs/reference/access-authn-authz/extensible-admission-controllers.md

```diff
@@ -1236,17 +1236,52 @@ object.
 
 ### Avoiding deadlocks in self-hosted webhooks
```

Comment:
Maybe this scenario can also be incorporated: https://github.com/kubernetes/enhancements/pull/5068#discussion_r1945419681

Reply:
This PR feels like it's at danger of not converging, I'd rather not add anything more to it.